jams/core.py

```diff
@@ -1016,10 +892,10 @@ def trim(self, start_time, end_time, strict=False):
         # We do this rather than copying and directly manipulating the
         # annotation' data frame (which might be faster) since this way trim is
         # independent of the internal data representation.
-        for idx, obs in self.data.iterrows():
+        for obs in self.data.obs:
```

Comment:
Are you talking about `Annotation` or `AnnotationData`?

The latter is iterable (wrapping the sorted list); the former is not and wasn't previously.  We could make it iterable, but that sounds like a change better suited to a separate PR.